ibis/backends/base/sql/__init__.py

```diff
@@ -200,6 +200,7 @@ def _log(self, sql: str) -> None:
         This method can be implemented by subclasses. The logging happens
         when `ibis.options.verbose` is `True`.
         """
+        print(sql)
```

Comment:
You should be able to achieve this by setting `ibis.options.verbose = True`. Does that not work?